regions/src/main/java/software/amazon/awssdk/regions/providers/AwsRegionProviderChain.java

```diff
@@ -29,6 +30,7 @@
  *
  * Throws an {@link SdkClientException} if region could not be find in any of the providers.
  */
+@SdkProtectedApi
```

Comment:
Why is this not public?

Reply:
We can make it public for GA, but we haven't reviewed it as a public API yet.